src/models/meals.model.ts

```diff
@@ -0,0 +1,23 @@
+export enum MealCategories {
+  BREAKFAST = 'Breakfast',
+  LUNCH = 'Lunch',
+  DINNER = 'Dinner',
+}
+
+export type MealText = string;
+export type Category = string;
+
+export interface AppButton {
```

Comment:
although this doesn't have anything to do with `meals` (name of the file) - I'll zorem on this one and not ask to move to a separate file

Reply:
can i do a file called models.ts which will contain everything? and if i see it's getting big so i'll separate.

Reply:
please don't - why not get used to working neat? it's ok to have a file with one model even

Reply:
k i'll make them separate